drivers/nvme/host/core.c

```diff
@@ -2719,8 +2719,21 @@ static struct nvme_subsystem *__nvme_find_get_subsystem(const char *subsysnqn)
 		return NULL;
 
 	list_for_each_entry(subsys, &nvme_subsystems, entry) {
+		bool valid_ctrl = false;
+		struct nvme_ctrl *ctrl;
+
 		if (strcmp(subsys->subnqn, subsysnqn))
 			continue;
+		if (!list_empty(&subsys->ctrls)) {
```

Comment:
In case of empty list seems you could just "continue" to reduce indentation.  Otherwise looks good to me.

Reply:
Thanks. Updated!